manifests/09_service_monitor.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  creationTimestamp: "2019-09-03T14:20:41Z"
+  generation: 1
+  labels:
+    name: marketplace-operator
+  name: marketplace-operator
```

Comment:
Same comment about name as above.

Reply:
We can call it `marketplace-operator-metrics` if you'd like.

Reply:
Leaving this as marketplace-operator given that a [similar approach](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/ocp/manifests/0.10.0/0000_50_olm_19-service-monitor.yaml#L7) was taken by OLM.